app/views/layouts/_toast.html.slim

```diff
@@ -0,0 +1,17 @@
+.toast-container.position-fixed.bottom-0.end-0.p-3
+  - if !!alert
```

Comment:
'if !!alert' is the same as 'if alert'??

Reply:
It's a JS habit :) In Ruby we do nor need convert value to compare it.

Reply:
Please,  fix it :)